src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -185,16 +199,34 @@ export class MeetingManager implements AudioVideoObserver {
     configuration: MeetingSessionConfiguration,
     deviceLabels: DeviceLabels | DeviceLabelTrigger = DeviceLabels.AudioAndVideo,
   ): Promise<any> {
-    const { logger: inputLogger, simulcastEnabled, videoDownlinkBandwidthPolicy } = this.config;
-    const logger = inputLogger ? inputLogger : this.createLogger(configuration);
-
+    const {
+      simulcastEnabled,
+      logger: configLogger,
+      logLevel,
+      postLogConfig,
+      videoDownlinkBandwidthPolicy
+    } = this.meetingManagerConfig;
+    
     if (simulcastEnabled) {
+      this.simulcastEnabled = simulcastEnabled;
       configuration.enableUnifiedPlanForChromiumBasedBrowsers = true;
       configuration.enableSimulcastForUnifiedPlanChromiumBasedBrowsers = true;
-    };
-
+    }
+    if (configLogger) {
+      this.logger = configLogger;
+    } else {
+      this.logLevel = logLevel;
+      if (postLogConfig) {
+        this.postLoggerConfig = postLogConfig;
+      }
+    }
     if (videoDownlinkBandwidthPolicy) {
-      configuration.videoDownlinkBandwidthPolicy = videoDownlinkBandwidthPolicy;
+      this.videoDownlinkBandwidthPolicy = videoDownlinkBandwidthPolicy;
+    }
+    const logger = this.logger ? this.logger : this.createLogger(configuration);
+
+    if (this.videoDownlinkBandwidthPolicy) {
+      configuration.videoDownlinkBandwidthPolicy = this.videoDownlinkBandwidthPolicy;
```

Comment:
Could we just set this.videoDownlinkBandwidthPolicy and not use them? In v3, we probably just remove them anyway so not using them would make remove them later easier.

Reply:
Ok makes sense. Have update the code.